xml/System.Net.Http/HttpClient.xml

```diff
@@ -67,7 +67,29 @@ public class GoodController : ApiController
   
 ```  
   
-   
+The <xref:System.Net.Http.HttpClient> is a high-level API that wraps the lower-level functionality available on each platform where it runs.
+
+On each platform, <xref:System.Net.Http.HttpClient> tries to use the best available transport:
+
+| **Host/Runtime**  | **Backend**                                                                             |
+| ----------------- | --------------------------------------------------------------------------------------- |
+| Windows/.NET      | <xref:System.Net.HttpWebRequest>                                                        |
```

Comment:
".NET" could apply to multiple .NETs, e.g. .NET Framework, .NET Core, etc.  We should be specific, with whatever terminology is appropriate these days, I assume ".NET Framework".

Reply:
Agreed